Add Backstage intro page (#2238)
giantswarm/docs: src/content/overview/backstage-portal/_index.md

---

> title: Backstage portal

Reviewer: IMO, I will call it a developer portal cause it is more than a Backstage instance; we serve some special plugins, we operate it,...

Reviewer: we dont use in any action the name of the tooling but what general one

Author: All right, fixed to align

> description: Backstage is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.

Reviewer: ```suggestion
description: Backstage is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so engineers using your platform find all the information they need in the right place.
```

weight: 30
menu:
  principal:
    parent: overview
    identifier: overview-backstage-portal
last_review_date: 2024-07-01
owner:
  - https://github.com/orgs/giantswarm/teams/team-honeybadger
---

> # Backstage portal

Reviewer: ```suggestion
```
Not needed

> [Backstage](https://backstage.io/) is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.

Reviewer: I would like to link to the CNCF page for it to highlight our commitment to the initiative. I do the same with others.
https://www.cncf.io/projects/backstage/

> Depending on where you are in your journey towards your own Backstage portal, Giant Swarm can assist you in various ways. We provide a pre-configured Backstage instance as a starting point to customers who don't run their own instance yet. We also provide plugins that can be used to extend the functionality of your Backstage instance.

Reviewer: ```suggestion
Giant Swarm can assist you in various ways depending on where you are in your journey towards your own developer portal. We provide a pre-configured Backstage instance as a starting point for customers who haven't run their own instances yet. We also provide plugins that can be used to extend the functionality of your Backstage instance.
```
